src/bootstrap/flags.rs

```diff
@@ -27,6 +29,16 @@ impl Default for Color {
     }
 }
 
+impl From<Color> for ColorChoice {
```

Comment:
Since I'm adding termcolor anyway, it probably makes sense to remove Color and just use ColorChoice directly.